gps_waypoint_collector:
  ros__parameters:
    frequency: 20
    yaml_file_out: ""
    path_length: 50
    client_node_name: 'nav2_wp_follower_client'